### ISSUE TITLE
Deleteing an annotation should require CTRL-DEL

### ISSUE THREAD

**Issue by kerryland** [sumatrapdfreader/sumatrapdf#3723]
Currently an annotation can be deleted simply by pressing the `DEL` key when you are in the listbox, or by pressing `DEL` in the PDF view. This is catastrophic if you *think* that the text-entry field has focus, and that you are simply deleting a few characters from a comment.

This is especially heinous because there is no "undo" function, so you have just deleted an annotation that you can't get back, and aren't even sure what you have lost.

**Comment by GitHubRulesOK** [comment]
yes caught me out when i thought chubby fingers were aimed at end key !
